tmp.txt

```diff
@@ -0,0 +1 @@
+"<div id=\"app\"><h1>Frontend-Einkaufslisten-Bearbeitungs-Und-Erstellungsmaschine</h1> <div><div class=\"container px-lg-5\"><div class=\"row mx-lg-n5\"><div class=\"col-2 py-3 border bg-light\"><button type=\"button\" name=\"edit-button\" id=\"buttonEdit\" class=\"btn btn-primary\">Edit</button> <!----></div> <div class=\"col py-3 border bg-light\"><form class=\"text\"><p id=\"text\">Get A+ in SDF</p> <!----></form></div> <div class=\"col-2 py-3 border bg-light\"><button type=\"button\" name=\"delete-button\" id=\"buttonDelete\" class=\"btn btn-danger\">Delete</button></div></div><div class=\"row mx-lg-n5\"><div class=\"col-2 py-3 border bg-light\"><button type=\"button\" name=\"edit-button\" id=\"buttonEdit\" class=\"btn btn-primary\">Edit</button> <!----></div> <div class=\"col py-3 border bg-light\"><form class=\"text\"><p id=\"text\">Trashtalk the vue 'Getting started' guide</p> <!----></form></div> <div class=\"col-2 py-3 border bg-light\"><button type=\"button\" name=\"delete-button\" id=\"buttonDelete\" class=\"btn btn-danger\">Delete</button></div></div><div class=\"row mx-lg-n5\"><div class=\"col-2 py-3 border bg-light\"><button type=\"button\" name=\"edit-button\" id=\"buttonEdit\" class=\"btn btn-primary\">Edit</button> <!----></div> <div class=\"col py-3 border bg-light\"><form class=\"text\"><p id=\"text\">git commit -m 'this'</p> <!----></form></div> <div class=\"col-2 py-3 border bg-light\"><button type=\"button\" name=\"delete-button\" id=\"buttonDelete\" class=\"btn btn-danger\">Delete</button></div></div> <div class=\"row mx-lg-n5\"><div class=\"col-12 py-3 border bg-light\"><button type=\"button\" id=\"btnAdd\" class=\"btn btn-success\">Add</button></div></div></div></div></div>"
```

Comment:
What's this file? Is it related?

Reply:
This file was only used for debugging purposes and slipped into the repo on accident. To be removed